nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -187,7 +185,16 @@ public List<TableIdentifier> listTables(Namespace namespace) {
 
   @Override
   public boolean dropTable(TableIdentifier identifier, boolean purge) {
-    return client.dropTable(identifier, purge);
+    TableReference tableReference = parseTableReference(identifier);
```

Comment:
Don't we have the same issue with `renameTable`?

Reply:
Good catch. I saw it during debugging. But forgot to handle. 
Handled now.